Cargo.toml

```diff
@@ -10,7 +10,7 @@ description = "Library for Intel TDX"
 readme = "README.md"
 keywords = ["intel", "sgx", "tdx"]
 categories = ["os", "os::linux-apis", "hardware-support"]
-rust-version = "1.71"
+rust-version = "1.73"
```

Comment:
Any particular reason for this?

Reply:
I copied the sized vec code from libkrun and it needed a newer MSRV. I can re-write it if that's easier. 

Is there a reason why we have such an old version as the MSRV?

Reply:
No, that's fine, just wondering. We can probably go up to 1.86?